tests/build/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
+- simonbaird
+- lcarva
+- zregvart
+- joejstuart
+- robnester-rh
+- cuipinghuo
```

Comment:
Do you need to add `reviewers` section? for example
```
approvers:
- cuipinghuo
...
reviewers:
- cuipinghuo
....
```

Reply:
My understanding is that we don't need to add reviewers, as we have this configuration enabled: https://github.com/openshift/release/blob/master/core-services/prow/02_config/redhat-appstudio/e2e-tests/_pluginconfig.yaml#L39
So any github review approval will add LTGM if I understand it correctly.